components/Menu/structure.ts

```diff
@@ -103,7 +103,7 @@ const menu: MenuCategoryProps[] = [
       {
         icon: "stack",
         title: "Documentation",
-        description: "Developer documentation for using Teleport",
+        description: "Installation and Teleport user guides",
         href: "/docs/",
```

Comment:
Is this link right still? It 404s on Vercel

Reply:
Updated to an absolute url